.github/workflows/release.yml

```diff
@@ -156,6 +127,7 @@ jobs:
             build_pip_pkg
           bazel-bin/build_pip_pkg artifacts
       - uses: actions/upload-artifact@v1
+        if: github.event_name == 'release' || matrix.python-version == '3.5'
```

Comment:
Wondering why we would need to upload-artifact for CI testing?

Reply:
To test the small job where we gather actifacts. If you see a simpler way, I'm open to ideas.

Reply:
Gotcha. Better to test that than not test I suppose, but I do think the most likely cause of a failure there would be timeout or something.